globus_sdk/config/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .env_vars import get_environment_name, get_http_timeout, get_ssl_verify
+from .environments import get_config_by_name, get_service_url
+
+__all__ = (
```

Comment:
I believe that `__all__` just needs to be an iterable, though I could be mistaken.
I used to use pylint occasionally to look for interesting maybe-bugs, and it will complain when `__all__` is a mutable type, so I developed the habit of writing it as a tuple.

It really doesn't matter, since `__all__` is for wildcard imports and you shouldn't ever be using those. But it's a nice way of satisfying flake8, and "declaring things".

I have set `per-file-ignores = __init__.py:F401` in flake8 config in the past, and wouldn't mind doing that + dropping all of the `__all__` lists/tuples from the SDK.